strongbox-data-service/src/main/java/org/carlspring/strongbox/config/DataServiceConfig.java

```diff
@@ -107,15 +78,16 @@ public EntityManagerFactory entityManagerFactory()
 
         LocalContainerEntityManagerFactoryBean result = new LocalContainerEntityManagerFactoryBean();
         result.setJpaPropertyMap(jpaProperties);
-        result.afterPropertiesSet();
+        result.setDataSource(dataSource);
+        result.setPersistenceUnitName("strongbox-PU");
```

Comment:
shouldn't this and below be defined in `persistence.xml`?

Reply:
We can drop the xml (persistence.xml precisely) and configure persistence unit programmatically this way by using Java config and Spring features. I prefer manipulating java code than xml.

Reply:
+1: I agree. Wherever possible, we should do the configuration programatically.

Reply:
Yes, it is removed in this PR :)